pages/charts.tsx

```diff
@@ -55,3 +57,13 @@ export default function Charts() {
     </main>
   )
 }
+
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+export async function getStaticProps({ locale }) {
+  return {
+    props: {
+      ...(await serverSideTranslations(locale)),
```

Comment:
It's kind of inconvenient to have to put this everywhere -- I wonder if there's a way around it